FEM/fem_ele_vec.cpp

```diff
@@ -1954,10 +1950,8 @@ void CFiniteElementVec::LocalAssembly_continuum(const int update)
         JFNK = true;
 #endif
     //
-    ThermalExpansion = 0.0;
-    // Thermal effect
-    if (smat->Thermal_Expansion() > 0.0)
-        ThermalExpansion = smat->Thermal_Expansion();
+    const double ThermalExpansion = (smat->Thermal_Expansion() > 0.0) ?
```

Comment:
- in which case, one has a negative thermal expansion coeff? 
- If the coeff can be negative, we can make it shorter  with `ThermalExpansion = std::max(smat->Thermal_Expansion(), 0.0);`

Reply:
Removed this ternary assignment.

The only place for that case is for output the solid parameters, see
https://github.com/ufz/ogs5/blob/9ca6118ccb09973140626d17ecbb3b06b96c3d17/FEM/rf_msp_new.cpp#L8569